packages/component-library/components/Notification/_styles.scss

```
@@ -71,42 +74,45 @@ $ca-notification-slide-right: transform $ca-duration-slow ease-out;
   // Types
 
   &%ca-notification---affirmative {
-    background: $kz-color-seedling-100;
-    border-color: $kz-color-seedling-300;
-    color: $kz-color-wisteria-800;
+    background: $kz-var-color-seedling-100;
+    border-color: $kz-var-color-seedling-300;
+    color: $kz-var-color-wisteria-800;
 
     &%ca-notification---global {
-      background-color: $kz-color-seedling-200;
+      background-color: $kz-var-color-seedling-200;
     }
   }
 
   &%ca-notification---informative {
-    background: $kz-color-cluny-100;
-    border-color: $kz-color-cluny-300;
-    color: $kz-color-wisteria-800;
+    background: $kz-var-color-cluny-100;
+    border-color: $kz-var-color-cluny-300;
+    color: $kz-var-color-wisteria-800;
 
     &%ca-notification---global {
-      background-color: $kz-color-cluny-200;
+      background-color: $kz-var-color-cluny-200;
     }
   }
 
   &%ca-notification---cautionary {
-    background: $kz-color-yuzu-100;
-    border-color: $kz-color-yuzu-400;
-    color: $kz-color-wisteria-800;
+    background: $kz-var-color-yuzu-100;
+    border-color: $kz-var-color-yuzu-400;
+    color: $kz-var-color-wisteria-800;
 
     &%ca-notification---global {
-      background-color: $kz-color-yuzu-300;
+      background-color: var($kz-var-id-color-yuzu-200, $kz-var-color-yuzu-300);
```

Add a quick comment to make it easier to find the cleanup

```suggestion
      // Use Yuzu200 for Heart theme, and Yuzu300 for zen (the default fallback)
      background-color: var($kz-var-id-color-yuzu-200, $kz-var-color-yuzu-300);
```